apps/src/blockTooltips/DropletFunctionTooltip.html.ejs

```diff
@@ -1,13 +1,15 @@
 <div class="function-name">
   <% if (signatureOverride) { -%>
     <%= signatureOverride %>
+  <% } else if (isProperty) { -%>
+    <% if (tipPrefix) { %><%= tipPrefix %><% } %><%= functionName %>
```

Comment:
Maybe I don't understand the more general purpose of `tipPrefix` then.  Don't all the current prefix values end with a dot?